### DISCUSSION TITLE
Authorising with the demo (Yet Another Demo Question)

### DISCUSSION THREAD

**Discussion by martynia** [DIRACGrid/diracx#238] [Developer discussions]
I have 2 demo installations, one is an older one which we did during the last DiracX hackhathon at CERN based on instructions which once existed in diracx-charts and a newer one which includes diracx-web. Both start fine,. I remember that I had to tick a _vo:diracAdmin_ box under "Available authorizations" but this box does not exist anymore. If I ignore this and click "authorise" I'm getting:
`{"detail":"No vo scope selected, available values: 'vo:diracAdmin'"}`

What could be a problem ?

**Comment by chaen** [comment]
Can you put screenshots of what you see ?

**Comment by martynia** [comment]
Hi,
I'm unable to reproduce the problem.

The problem first appeared when I used my laptop at home behind NAT. I haven't saved a screenshot back then. I tried the demo in the College, both on eduroam and a public desktop IP address and it worked fine. The checkbox is there (the fourth from the top). I'm back at home, started the demo and the box is still there.
So for the time being it looks that I have to have my vision checked ;-(

**Reply by chaen** [reply]
Let's consider it solved then :-)

**Reply by chaen** [reply]
Btw, this is more of an issue than a discussion to open :-)
